tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "target": "es2019",
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
`es2020` works too...

Reply:
https://stackoverflow.com/a/59787575/77199 suggests that `target` should remain `es2019` but can add `"lib": ["es2020"]`.